client/@/css/app.css

```diff
@@ -87,8 +88,10 @@ nav .icon, nav a
 }
 
 ul.comments {
-  padding-bottom: 0;
   margin-bottom: 0;
+  margin-top: calc(var(--padding) * 4);
+  padding-bottom: calc(var(--padding) * 2);
+  padding-left: calc(var(--padding) * 2);
```

Comment:
I'm afraid this doesn't work well with _rtl_ browsers/OS ... and I've been thinking about this issue, but couldn't find a way to properly address comments indeed.

Note that nothing uses `left` or `right`, it'e eventually `start` or `end` with flexbox, which plays well with _rtl_ systems.

Reply:
P.S. truth to be told, I could overwrite the rule but I'm not sure if `:dir(rtl)` works well in the wild, and MDN suggests it doesn't https://developer.mozilla.org/en-US/docs/Web/CSS/:dir

Reply:
Oh right, language direction is an interesting problem I didn't consider. I'll give it some thought too.

Reply:
I added `[dir="rtl"]` and that seems to work on the browsers I tested. See what you think.

Reply:
the issue is that browsers running in a rtl OS don't add that attribute and nothing in the app adds it neither ... still thinking about a better way